gtsam.h

```diff
@@ -1977,6 +1983,35 @@ size_t symbol(char chr, size_t index);
 char symbolChr(size_t key);
 size_t symbolIndex(size_t key);
 
+namespace symbol_shorthand {
```

Comment:
was this tested? If so where?

Reply:
I tested this in an example. I want to port all the examples, so this PR is still technically WIP.